students/itsabhi/grid.py

```diff
@@ -0,0 +1,27 @@
+def printnewrow(size):
```

Comment:
Python convention is to use 'snake case' such as print_new_row for naming functions and variables.